configs/template_config.yaml

```diff
@@ -81,6 +81,14 @@ tokenized_dataset_path: "<YOUR_PATH_HERE>/data/tokenized_dataset/c4"
 # TensorBoard Path (str): Path for where TensorBoard files go, optional
 tboard_path: "<YOUR_PATH_HERE>/data/runs"
 
+# This is where you can specify what percentage of C4 data you want to use for training
+# IMPORTANT!!!!!
+# When set to true, you will need to go into dataset.py and edit the setup function in the DataModule class
+# Then instead of training from the train directory, you will change that to "portion_data" (About line 43)
+split_dataset: true
+split_dataset_percentage: 1
+train_model_data_path: "portion_data" # Set to "train" to use the full training directory for your model
```

Comment:
In keeping with the current format of `template_config.yaml`, there should be an individual comment above each configuration argument with the explanation. I'd also move the inline comment for `train_model_data_path` to such an above comment as well.